backend/voting/admin.py

```diff
@@ -51,7 +51,8 @@ class Media:
 
 
 @admin.register(RankSubmission)
-class RankSubmissionAdmin(admin.ModelAdmin):
+class RankSubmissionAdmin(AdminUsersMixin):
+    user_fk = "submission.speaker_id"
```

Comment:
```suggestion
    user_fk = "submission__speaker_id"
```

this fk goes to a query, doesn't access the object